### DISCUSSION TITLE
Feature Request: any plan to support element-wise aggregation on vector(array) data?

### DISCUSSION THREAD

**Discussion by Rhett-Ying** [duckdb/duckdb#13467] [Ideas]
I am wondering if there's any plan for vector data support in DuckDB. The operations includes element-wise aggregation such as `sum`, `max`, and vector similarity search such as `TopK(target_vector)`.

**Comment by Maxxen** [comment]
https://duckdb.org/docs/sql/data_types/array.html

**Comment by Rhett-Ying** [comment]
@Maxxen Hi, could you show me how to run element-wise sum? any specific function like `array_sum()`?

**Reply by Rhett-Ying** [reply]
`list_aggregate(values, 'sum')` ?

**Reply by Rhett-Ying** [reply]
above one generates single value, just sum all values in list instead of element-wise
